src/api/lib/storage/backends/consul.js

```diff
@@ -0,0 +1,109 @@
+var Storage = require('../');
+var consul = require('consul');
+var path = require('path');
+
+module.exports = ConsulBackend;
+
+function ConsulBackend(options) {
+	options = options || {};
+	this._prefix = options.prefix || 'billimanjaro';
+	delete options.prefix;
```

Comment:
So I can send the remaining options, without the prefix (just in case for backwards compatibility), to the consul client.